docs/.wordlist.txt

```diff
@@ -1,77 +1,93 @@
+ACR
```

Comment:
are we now sorting ignoring case?

Reply:
https://github.com/canonical/sphinx-docs-starter-pack/blob/main/.wordlist.txt

Reply:
I did a previous PR with a case sensitive sort and removed lots of duplicates, just asking if this was on purpose, and it's fine, not asking for a change or anything

Reply:
right. I see the diff. I think that re-arrangement was committed by @SamirPS , so I'm not 100% sure if there's is an underlying reason. I'm also fine either way...

Reply:
no reason, it's was just a sort